input/fsh/CodeSystem-Omim.fsh

```diff
@@ -0,0 +1,508 @@
+CodeSystem: Omim
```

Comment:
Why don't we unify the CodeSystem resource name? I am suggesting to either:

- Spell it out with camel casing like `OnlineMendelianInheritanceInMan` as we've did for HPO and change the file name accordingly; or
- Let's use acronyms like `CodeSystem: OMIM` and modify the file name accordingly

With either option, let's apply the same rule for the other CodeSystem resources.

Reply:
Any thoughts on which would be more recognizable? HPO / OMIM are used a lot by folks I've worked with. Also OnlineMendelianInheritanceInMan is pretty long. Seems like acronyms may be more helpful if someone were to be looking at files as well as resources used within the server

Reply:
Yeah, and I don't the file names to be too verbose. Plus,. the full ontology name will be put at `Title`. So, I am down for the idea of using an acronym for the file name as well as the CodeSystem name. So, for HPO and OMIM, I am suggesting:

- HPO
  - File name: `CodeSystem-HPO.fsh`
  - CodeSystem name: `CodeSystem: HPO` 
- OMIM
  - File name: `CodeSystem-OMIM.fsh`
  - CodeSystem name: `CodeSystem: OMIM`

Reply:
Done